src/Menu/Menu.js

```diff
@@ -0,0 +1,37 @@
+import { useEffect, useState } from 'react'
+import './Menu.scss'
+
+const Menu = (props) => {
+    const [foodItems, setFoodItems] = useState(null)
+
+    useEffect(() => {
+        if (props.restaurantDetails) {
+            setFoodItems(props.restaurantDetails.foodItems)
+        }
+    }, [props.restaurantDetails])
```

Comment:
you actually don't need to store this as state at all, instead you can just use the prop everywhere. e.g. instead of
`foodItems?.map(...)` you can do `props?.restaurantDetails.foodItems.map(...)`

Reply:
Done!